app/ide-desktop/lib/dashboard/src/authentication/src/dashboard/components/templates.tsx

```diff
@@ -123,13 +144,7 @@ const Templates: React.FC<TemplatesProps> = ({ onChange }) => {
         <div className="bg-white">
             <div className="mx-auto py-2 px-4 sm:py-4 sm:px-6 lg:px-8">
                 <div className="grid grid-cols-1 gap-2 sm:grid-cols-3 lg:grid-cols-4 xl:grid-cols-5">
-                    {templates.map(template => (
-                        <TemplateRender
-                            key={template.title}
-                            template={template}
-                            onChange={onChange}
-                        />
-                    ))}
+                    <TemplatesRender templates={templates} onChange={onChange} />
```

Comment:
actually, i'm not sure templates should be a separate component
in other files i think we normally just create a variable when the JSX gets too complex... what do you think?

Reply:
TemplatesRender focuses on rendering and click interactions, which may become more complex later.
So I'm splitting it up here for now.
If it becomes more complex, we can move it to a new file.